### ISSUE TITLE
got it working. great stuff! thank you

### ISSUE THREAD

**Issue by ficshelf** [vladka/openshift-meteor-leaderboard-customNode#1]
by the way.. do you freelance? might need a spare set of hands!

**Comment by vladka** [comment]
Yes, I'm freelance in Czech republic (middle europe), but if I a have possibility to work in english speaking country, I can, because I need to improve my English :-)

**Comment by ficshelf** [comment]
Hi Vladka

why don't you get in touch here: martin@ficshelf.com  via email or hangouts
so we could discuss it further?

kind regards
